### DISCUSSION TITLE
GUID and Atom feed/item ID

### DISCUSSION THREAD

**Discussion by mainrs** [Podcastindex-org/podcast-namespace#622] [Q&A]
Hello! This namespace provides a GUID for the podcast itself as well as each item in it. The Atom feed's items should also have a unique ID each.

If I publish my podcast using the Atom feed format, would I set these properties to the same value? What is the go-to approach here?

**Comment by Marzal** [comment]
https://podcasters.apple.com/4115-technical-updates-for-hosting-providers

> Atom feeds
> Update: No longer supported
> New shows in Atom feed format will no longer be accepted and any remaining shows will no longer be updated.

Atom feed format it's not supported on Apple Podcast since 2022, so I would not recommend using it.

This namespace support GUID at the channel level, the item's GUIDs are from the RSS spec. And yes, I would set those with the same value if there is only one feed.
